.gitignore

```diff
@@ -54,3 +54,6 @@ xgboost
 
 # Docs
 node_modules
+
+mindsdb/share/man/man1/isympy.1
```

Comment:
These entries in the .gitignore do not seem to be general, but rather specific to your work environment. Let's remove them.